catalog/plugin/catalog-plugin-pdf/pom.xml

```diff
@@ -0,0 +1,56 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>ddf.catalog.plugin</groupId>
+        <artifactId>plugin</artifactId>
+        <version>2.9.0-SNAPSHOT</version>
+    </parent>
+
+    <name>DDF :: Catalog :: Plugin :: PDF</name>
```

Comment:
Just plugins seemed like the most fitting place since there aren't content/plugins (yet). Though I'm open to putting it wherever. Would you prefer a catalog/content/plugins?